src/pages/guidelines/color/implementation.mdx

```diff
@@ -0,0 +1,546 @@
+can include contextual tokens in their specs and redlines.
+
+<Row>
+<Column colLg={8}>
+
+![Component spec example using contextual tokens](images/color-implementation-contextual-3.png)
+
+</Column>
+</Row>
+
+### Using the layer component
+
+The layer component is used with the contextual tokens to implement the layering
+model in code. The layer component and contextual tokens are essentially a short
+cut for development because the layer component will apply the correct values to
+a single component without having to develop explicit hard coded variants.
+
+<InlineNotification>
+
+**Coming soon:** Developer documentation is currently be worked on. Check back
+at a later date for more information.
+
+</InlineNotification>
+
+## Inline theming
+
+Inline theming is used when a section of a UI needs to have a different theme
+from the rest of the page. Inline theming allows themes to be nested within each
+other without needing custom styles or overrides. In product, a common use-case
+for inline theming is applying a contrasting theme to UI Shell and side panels.
+This is especially common in light themed products or light modes. For example,
+the majority of a page can use the White theme while the UI Shell and the left
+side panel uses an inline Gray 100 theme. This type of inline theme pairing
+creates a high contrast moment that adds emphasis and focus to a work flow.
+
+You should only need to use inline theming for major shifts in color, like high
+contrast moments. The more subtle transitions of color in a product are handled
+within each theme through the layering model tokens. It is unlikely that you'll
+need to inline a White theme within the Gray 10 theme or the Gray 90 theme
+within the Gray100 theme.
+
+<Row>
+<Column colLg={8}>
+
+![Example of inline themes](images/color-implementation-inline-1.png)
+
+</Column>
+</Row>
+
+#### How it works
+
+If your product is using the Carbon color tokens and themes then a section of
+the page can easily be targeted to call a different theme without having to use
+new tokens or manually change values. For the inline themed section simply use
+the same Carbon tokens you were using for the rest of the page, then using the
+theme component assign the whole zone the new theme.
+
+When designing for inline theming you'll need to pull assets from the other
+theme library files (or use the
+[theme switcher](https://github.com/IanKitchen-IBM/carbon-theme-switcher) Sketch
+plug-in). Assets will have the same tokens names across libraries but will show
+different values. Then specify in the design deliverables which sections of the
+page will be using an inline theme.
+
+<Row>
+<Column colLg={8}>
+
+![Example of inline themes](images/color-implementation-inline-2.png)
+
+</Column>
+</Row>
+
+### Using the theme component
+
+<InlineNotification>
+
+**Coming soon:** Developer documentation is currently be worked on. Check back
```

Comment:
Theme dev docs for v11: https://carbon-react-next.netlify.app/?path=/docs/components-theme--default

Reply:
Again to we want to embed any of these docs on this page or just always link out to storybook?